osf/external/gravy_valet/translations.py

```diff
@@ -23,6 +23,12 @@ def make_ephemeral_user_settings(gv_account_data, requesting_user):
         active_user=requesting_user,
     )
 
+def get_links(node_guid, requesting_user=None):
```

Comment:
Same, we already have it merged from https://github.com/CenterForOpenScience/osf.io/pull/11113/files#diff-206d6f4977b86acd4f6bff7aeca1d4c23e54271d998524873afc55805974fdc3